ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml

```diff
@@ -1,5 +1,15 @@
 canonical_go_repository: github.com/redhat-developer/service-binding-operator
 
+base_images:
```

Comment:
Are the `redhat-developer/service-binding-operator` changes intentional for a PR that otherwise deals with `coderead-toolchain/toolchain-common`?

Reply:
oh, let me check that

Reply:
I got some incoming changes that I merged from `master` as there were conflicts. Let me redo this PR as I suspect I did something wrong :/ My apologies.

Reply:
Looks like there's something wrong with the PR on GitHub: locally, my changes for this PR are:
```
modified:   cluster/ci/config/prow/config.yaml
modified:   cluster/ci/config/prow/plugins.yaml

new: ci-operator/config/codeready-toolchain/toolchain-common/
new: ci-operator/jobs/codeready-toolchain/toolchain-common/
```

I suggest I close this one and open a new one

Reply:
Ah, I see: this PR includes https://github.com/openshift/release/pull/4206/commits/843b4dd240b77d2e7fa424892a68445bb0410789 but it should not! Let me fix that!

Reply:
@petr-muller I fixed the PR by "force pushing" a new version with the single commit that was meant to be part of the PR and reviewed. Sorry for the bad operation :/